### PR TITLE
feat: display correlationId to the rendered html

### LINKED ISSUE #108: Display correlationId in generated doc
#### Reason/Context
Please try answering few of those questions
- Why we need this improvement?

In my case, i'm using Kafka to do Events exchanges. I use [Request-reply](https://www.enterpriseintegrationpatterns.com/patterns/messaging/RequestReply.html) for exchange with the [CorrelationIdentifier pattern](https://www.enterpriseintegrationpatterns.com/patterns/messaging/CorrelationIdentifier.html)

- How will this change help?

My producers and consumers must see in the doc how the correlationId is generated (based on which fields in the contract) to allow filtering on correlationId for example.

- What is the motivation? 

#### Description
Please try answering few of those questions
- What changes have to be introduced?

Display informations about `CorrelationId` in `message` documentation

- Will this be a breaking change?

no
- How could it be implemented/designed?

UI proposal :

image

Code Snippet : 

```
<div class="bg-grey-lighter rounded p-4 mt-2">
  <div class="text-sm text-grey-darker mb-2">Correlation ID<span class="border text-orange rounded text-xs ml-3 py-0 px-2">$message.header#/CLIENT_CODE $message.payload#/PRODUCT_ID</span>
    
  </div>
  <p class="text-grey-dark text-sm">This correlation ID is used for message tracing and messages correlation.
            This correlation ID is generated at runtime based on Client ID / Product ID and costing request timestamp</p>
  
</div>
```

- the CorrelationID `location` attribut is describe into the `text-sm text-grey-darker mb-2` CSS classes

### PR DESCRIPTION
**Pull request by M3lkior**
Close #108

### CODE REVIEW

**Comment by M3lkior** [comment]
Here is the display for my enterprise project : 
image

And the generated based on the dummy.yml 👍 
image

image

File: package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "lint": "echo \"No linter specified yet\"",
     "test": "ava",
     "release": "semantic-release",
-    "develop": "ag https://raw.githubusercontent.com/asyncapi/generator/v1.0.0-rc.12/test/docs/dummy.yml ./ -o test/output --force-write --watch-template",
+    "develop": "ag https://raw.githubusercontent.com/asyncapi/generator/master/test/docs/dummy.yml ./ -o test/output --force-write --watch-template",
```
Comment (derberg) [inline package.json:25]:
```suggestion
    "develop": "ag https://raw.githubusercontent.com/asyncapi/generator/v1.1.5/test/docs/dummy.yml ./ -o test/output --force-write --watch-template",
```

there is a certain level of risk that the version on master gets updated and before it is supported by html-template, someone can have issues

File: README.md [hunk 2]
```diff
@@ -33,6 +33,11 @@ npm install -g @asyncapi/generator
     >**NOTE:** If you have to modify the [`dummy.yml`](https://github.com/asyncapi/generator/blob/master/test/docs/dummy.yml) file to develop your features, open a PR with the changes in the [asyncapi/generator](https://github.com/asyncapi/generator) repository.
 
 1. Generate output with watcher enabled: `npm run develop`.
+   
+   >**NOTE:** If your changes are not visible, this is maybe because the `ag` use the already installed `html-template` so you should use the `--install` option
+   > * run `ag https://raw.githubusercontent.com/asyncapi/generator/master/test/docs/dummy.yml ./ -o test/output --install --force-write --watch-template`
```
Comment (derberg) [inline README.md:38]: better would be to add a new script for this, like `develop:install`

**Review by derberg — CHANGES_REQUESTED** [review]
@M3lkior great first PR! I only made some suggestion to the version of the dummy file we use for development and a suggestion for another script

File: package.json (hunk 1 above)
Comment (M3lkior) [reply package.json:25]: ok, i will rollback this

File: README.md (hunk 2 above)
Comment (M3lkior) [reply README.md:38]: what about 
```   
"//develop:install": "Force ag to install your local html-template source code",
"develop:install": " npm run develop -- --install",
```
@deberg ?

File: package.json (hunk 1 above)
Comment (M3lkior) [reply package.json]: done

File: README.md (hunk 2 above)
Comment (derberg) [reply README.md]: @M3lkior perfect

Comment (M3lkior) [reply README.md]: i pushed this ; let me know if it is ok for you

File: package.json [hunk 3]
```diff
@@ -22,7 +22,9 @@
     "lint": "echo \"No linter specified yet\"",
     "test": "ava",
     "release": "semantic-release",
-    "develop": "ag https://raw.githubusercontent.com/asyncapi/generator/v1.0.0-rc.12/test/docs/dummy.yml ./ -o test/output --force-write --watch-template",
+    "develop": "ag https://raw.githubusercontent.com/asyncapi/generator/v1.1.5/test/docs/dummy.yml ./ -o test/output --force-write --watch-template",
+    "//develop:install": "Force ag to install your local html-template source code",
```
Comment (derberg) [inline package.json:26]: an interesting approach to document a script, I've never seen it before

Comment (M3lkior) [reply package.json:26]: yeah, it is a good trick ; i take this of my REACT experience ;)

**Comment by derberg** [comment]
@M3lkior all good from my side, you just need to resolve conflicts now

**Comment by M3lkior** [comment]
> @M3lkior all good from my side, you just need to resolve conflicts now

it seems ok now !

**Comment by derberg** [comment]
@M3lkior you had conflicts because of this PR https://github.com/asyncapi/html-template/pull/127/files that introduced tailwind v2. Please have a look and adjust the css classes accordingly, so basically no `dark` or `lighter` anymore

**Comment by M3lkior** [comment]
> @M3lkior you had conflicts because of this PR https://github.com/asyncapi/html-template/pull/127/files that introduced tailwind v2. Please have a look and adjust the css classes accordingly, so basically no `dark` or `lighter` anymore

Oups ^^ it is done

**Review by derberg — APPROVED** [review]

**Comment by derberg** [comment]
@all-contributors please add @M3lkior for code

**Pull request merged**